src/truncated/normal.jl

```diff
@@ -6,8 +6,9 @@ The *truncated normal distribution* is a particularly important one in the famil
 We provide additional support for this type with `TruncatedNormal` which calls `Truncated(Normal(mu, sigma), l, u)`.
 Unlike the general case, truncated normal distributions support `mean`, `mode`, `modes`, `var`, `std`, and `entropy`.
 """
```

Comment:
Thought: This doc string is valuable, we should preserve it at the end of the deprecation cycle.

Reply:
yes, I was thinking of defining a new type `TruncatedNormal`, which would implement these methods and keep the doc

Reply:
Let's postpone this problem just not forget it.